script_exporter.go

```diff
@@ -47,6 +48,7 @@ func runScript(script *Script) error {
 	defer cancel()
 
 	bashCmd := exec.CommandContext(ctx, *shell)
+	bashCmd.Env = append(os.Environ(), fmt.Sprintf("TARGET=%s", script.Target))
```

Comment:
What security concerns are there from an adversary injecting arbitrary text into the script's environment?